recordedit/form.controller.js

```diff
@@ -277,6 +285,7 @@
                             submit();
                         });
                     } else {
+                        // TODO: this needs to be removed and a catch added here
                         vm.showSubmissionError(exception);
                         vm.submissionButtonDisabled = false;
                     }
```

Comment:
The above code block can be removed in favor of a .catch() that will catch rejected promises and errors.